src/components/tooltip/Tooltip.js

```diff
@@ -115,7 +115,7 @@ class Tooltip extends React.Component<Props, State> {
         this.state = { isShown: !!props.isShown, wasClosedByUser: false };
     }
 
-    tooltipID = uniqueId('tooltip');
+    tooltipID = uniqueId('bdl-Tooltip');
```

Comment:
does the ID need to be in the same format as the class?

Reply:
Good catch. Will fix